rsocket-core/src/main/java/io/rsocket/core/RSocketServer.java

```diff
@@ -262,10 +282,41 @@ public RSocketServer payloadDecoder(PayloadDecoder decoder) {
   public <T extends Closeable> Mono<T> bind(ServerTransport<T> transport) {
     return Mono.defer(
         new Supplier<Mono<T>>() {
-          ServerSetup serverSetup = serverSetup();
+          final ServerSetup serverSetup = serverSetup();
 
           @Override
           public Mono<T> get() {
+            int maxFrameLength = transport.maxFrameLength();
+
+            if (maxFrameLength > FRAME_LENGTH_MASK) {
+              return Mono.error(
+                  new IllegalArgumentException(
+                      "Configured maxFrameLength["
+                          + maxFrameLength
+                          + "] exits maxFrameLength limit "
```

Comment:
This is meant to be `exceeds`? Search and replace here and in RSocketConnector.

Reply:
yeah...